app/index.html

```diff
@@ -394,9 +394,22 @@ <h1 class="col-sm-offset-1 col-md-offset-2 col-lg-offset-3 text-red">FEATURES</h
         <div class="col-md-offset-3">
           <div class="col-sm-12 col-md-5">
             <h1>Analyze Your Own Stack</h1>
-            <p>
-              Select a manifest file to run through our analyzer and see your results here.
-            </p>
+            <div class="dropdown dropdown-kebab-pf">
+              <span>
+              Select a manifest file to run through our analyzer and see your results here.</span>
+              <span data-toggle="dropdown" class="pficon pficon-info"></span>
+              <div class="dropdown-menu infotip">
+                <div class="arrow"></div>
+                <ul class="list-group">
+                  <li class="list-group-item">
+                    The manifest is a special file that can contain information about the files packaged
```

Comment:
I think this needs rewriting as it doesn't convey much info right now. Perhaps Preeti can help?